server/Account.java

```diff
@@ -19,12 +19,12 @@ public String getAccountName() {
 		return accountName;
 	}
 
-	//add getter for opening balance
+	// Adding getter for opening balance.
```

Comment:
I'd say the comments in this file are not needed, as the methods are descriptive enough, but it's a nitpick